### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
### What I did

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

### How I did it

The misspellings have been reported at https://github.com/jsoref/hadolint/actions/runs/4171605857#summary-11322471647

### How to verify it

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/hadolint/actions/runs/4171605992#summary-11322471818

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -287,7 +287,7 @@ jobs:
 
   release:
     needs: [docker-release, os-binaries]
-    name: Create Github Release
+    name: Create GitHub Release
```
Comment (jsoref) [inline .github/workflows/release.yml:290]: brand

File: src/Hadolint/Formatter/Codeclimate.hs [hunk 2]
```diff
@@ -120,8 +120,8 @@ issueToFingerprintIssue i =
 formatResult :: (VisualStream s, TraversableStream s, ShowErrorComponent e) => Result s e -> Seq Issue
 formatResult (Result filename errors checks) = (errorToIssue <$> errors) <> (checkToIssue filename <$> checks)
 
-formatGitlabResult :: (VisualStream s, TraversableStream s, ShowErrorComponent e) => Result s e -> Seq FingerprintIssue
-formatGitlabResult result = issueToFingerprintIssue <$> formatResult result
+formatGitLabResult :: (VisualStream s, TraversableStream s, ShowErrorComponent e) => Result s e -> Seq FingerprintIssue
+formatGitLabResult result = issueToFingerprintIssue <$> formatResult result
```
Comment (jsoref) [inline src/Hadolint/Formatter/Codeclimate.hs:124]: it's possible things like these are APIs. If so, I can drop them...

**Review by jsoref — COMMENTED** [review]
Most corrections suggested by Google Sheets. All fault mine

**Comment by lorenzo** [comment]
thanks!

**Pull request merged**